tests/test_all_rules.py

```diff
@@ -231,8 +231,8 @@ def test_required_tags(self):
             'auditbeat-*': {'any': ['Windows', 'macOS', 'Linux']},
             'endgame-*': {'all': ['Endpoint Security']},
             'logs-aws*': {'all': ['AWS']},
-            'logs-endpoint.alerts-*': {'all': ['Endpoint Security']},
```

Comment:
This change breaks testing for all the Elastic "Endpoint Security" rules that have the tag `Endpoint Security`